front/src/components/Navigation/NavigationLinks/NavigationLinks.js

```diff
@@ -7,6 +7,20 @@ import { Fragment } from "react";
 const NavigationLinks = (props) => {
     const isAuthenticated = useSelector((state) => state.isAuthenticated);
 
+    let navLinks = null;
+    if (isAuthenticated) {
+        navLinks = (
+            <Fragment>
+                <NavigationLink link="/admin/competitions">
+                    Moje zawody
+                </NavigationLink>
+                <NavigationLink link="/logout">Wyloguj</NavigationLink>
+            </Fragment>
+        );
+    } else {
+        navLinks = <NavigationLink link="/login">Zaloguj się</NavigationLink>;
+    }
+
```

Comment:
I had something like this in mind:
isAuthenticated ? linksForLoggedUser() : linksForUnknownUser()